web/src/routes/archive/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+	import { goto } from "$app/navigation";
+	import Button from "$lib/button.svelte";
+	import List from "$lib/list.svelte";
+	import Panel from "$lib/panel.svelte";
+	import { getFriendlyName } from "$lib/poll";
+	import { polls } from "../../store";
+
+	$: archivedPolls = $polls?.filter((p) => p.isConcluded);
+</script>
+
+<svelte:head>
+	<title>Users</title>
+</svelte:head>
+<Panel title="Archive" headerIcon="inventory_2">
+    <p>
+        All polls, including those from over 30 days ago
+    </p>
+	<List items={archivedPolls} let:prop={poll}>
+		<li class="poll">
+			<p>{getFriendlyName(poll)}</p>
+            <p>{(new Date(poll.date)).toUTCString()}</p> <!-- todo: probably a better idea to have the right type in the first place -->
```

Comment:
Please raise an issue for this and tag it in the todo comment so it doesn't get banished to the shadow realm over time :p